alpha/declcfg/filter/packages.go

```diff
@@ -0,0 +1,50 @@
+package filter
+
+import (
+	"context"
+	"slices"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+)
+
+func NewPackageFilter(keepPackages ...string) Filterer {
+	packageSet := sets.New[string](keepPackages...)
+	return &packageFilter{keepPackages: packageSet}
+}
+
+type packageFilter struct {
+	keepPackages sets.Set[string]
+}
+
+func (f *packageFilter) FilterCatalog(ctx context.Context, fbc *declcfg.DeclarativeConfig) (*declcfg.DeclarativeConfig, error) {
```

Comment:
My eyes could be deceiving me, but looks like this function doesn't use the `ctx` parameter.